### PR TITLE
Use guzzle to fetch Jwt Web Keys

### PR DESCRIPTION
**Pull request by Hodgy**
| Q                | A
| ---------------- | ---
| Type             |  Feature
| Ticket           | VOL-2381

**Description of the change:**
Due to the use of a proxy for outbound requests within VOL we need to replace the call to fetch JWT Web Keys with guzzle to allow it go via the proxy.

### CODE REVIEW

**Comment by JoshuaLicense** [comment]
@Hodgy Looks good! 

Just a few tests that pass Guzzle into the constructor still:

```
tests/ResourceOwnerObjectReturnedFromMethodsTest.php
tests/ContractExceptionsAreThrownInsteadTest.php
tests/AuthenticateReturnsExpectedResponseTest.php
tests/AttributesAreProvidedInCorrectFormatTest.php
```

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "aws/aws-sdk-php": "^3.183.7",
     "dvsa/contracts": "^1.1",
     "firebase/php-jwt": "^5.2",
+    "guzzlehttp/guzzle": "^7.3",
```
Comment (JoshuaLicense) [inline composer.json:19]: Could we set as `^6.0 || ^7.0` if possible - MOT is in a transition stage to PHP 8.0 and we still use Guzzle 6 in various places.

Comment (sdh100shaun) [reply composer.json:19]: Also as would be a new release give it a version :-)

**Review by sdh100shaun — APPROVED** [review]

**Comment by sdh100shaun** [comment]
As always approval on grounds conversations resolved

**Comment by sdh100shaun** [comment]
7.2 action failure should not be an issue as 7.4 out on VOL as of tonight

**Comment by Hodgy** [comment]
> 7.2 action failure should not be an issue as 7.4 out on VOL as of tonight

Unfortunately it's being caused  by Guzzle 6 as i have it happening locally for 7.4

**Review by JoshuaLicense — APPROVED** [review]

**Comment by sdh100shaun** [comment]
> > 7.2 action failure should not be an issue as 7.4 out on VOL as of tonight
> 
> Unfortunately it's being caused by Guzzle 6 as i have it happening locally for 7.4

ah I did not dig into detail thanks for double-checking

**Pull request merged**